taxonomy.qmd

```diff
@@ -1353,7 +1352,7 @@ distributed processing in artificial networks.
 
 
 
-Unfortunately, however, at that time we did not have the compute and
+Unfortunately, however, at that time we did not have the computers and
```

Comment:
I do think the intended word is probably "compute", as in "computational resources" or "computational power"
(indeed computers were readily available then)